src/site/includes/support_resources_search_bar.drupal.liquid

```diff
@@ -0,0 +1,50 @@
+<div class="vads-u-border-top--1px vads-u-border-bottom--1px vads-u-border-color--gray-lighter vads-u-padding-y--2 vads-u-margin-bottom--3">
+  <form id="resources-support-search" method="get" action="/resources/search" id="search-resources-and-support">
+    <fieldset class="fieldset-input vads-u-margin--0">
+      <legend class="vads-u-font-size--md vads-u-font-family--serif">Search resources and support articles or all of VA.gov</legend>
+      <label class="vads-u-visibility--screen-reader" for="website-section">Website section to search</label>
+      <div class="form-radio-buttons">
+        <div class="radio-button vads-u-display--inline-block vads-u-margin-right--3">
+          <input type="radio" name="path" checked id="search-within-resources-and-support" value="/resources/search"/>
+          <label for="search-within-resources-and-support">
+            <span class="vads-u-visibility--screen-reader">Search within</span> Resources and support
+          </label>
+        </div>
+        <div class="radio-button vads-u-display--inline-block">
+          <input type="radio" name="path" id="search-all-of-va-dot-gov" value="/search"/>
+          <label for="search-all-of-va-dot-gov">
+            <span class="vads-u-visibility--screen-reader">Search</span> All of VA.gov
+          </label>
+        </div>
+      </div>
+    </fieldset>
+    <label class="vads-u-margin-top--1" for="query">Enter a keyword, phrase, or question</label>
+    <div class="vads-l-row">
+      <div class="vads-l-col--12 vads-u-flex--1 vads-u-width--auto">
+        <input type="text" required name="query" id="query" class="usa-input vads-u-max-width--100 vads-u-width--full vads-u-height--full vads-u-margin--0">
+      </div>
+      <div class="vads-l-col--12 vads-u-flex--auto vads-u-width--auto">
+        <button type="submit" class="usa-button vads-u-margin--0 vads-u-width--full vads-u-width--auto vads-u-height--full" style="border-top-left-radius: 0; border-bottom-left-radius: 0;">
+          <i class="fa fa-search"> </i> Search
+        </button>
+      </div>
+    </div>
+  </form>
+</div>
+<script>
+  (function() {
+    var form = document.getElementById('resources-support-search');
+    form.addEventListener('submit', function(event) {
+      var searchGlobally = document.getElementById('search-all-of-va-dot-gov');
+
+      if (searchGlobally.checked) {
+        form.action = '/search';
+      }
+
+      // Prevent the "path" form field from being included in the form data to keep the URL cleaner.
+      // That field is only used to decide where to submit the search.
+      searchGlobally.removeAttribute('name');
+      document.getElementById('search-within-resources-and-support').removeAttribute('name');
+    });
+  })();
+</script>
```

Comment:
Takes me back...

Reply:
oh snap!! good to know you can do this in liquid templates

Reply:
It's probably not something we should do constantly but I think it's fine for little snippets like this